src/Bicep.LangServer.IntegrationTests/HighlightTests.cs

```diff
@@ -119,6 +119,16 @@ private static DocumentHighlightKind GetExpectedHighlightKind(SyntaxBase syntax)
             }
         }
 
+        private Position GetPosition(SyntaxBase syntax, ImmutableArray<int> lineStarts)
```

Comment:
Added this method in the unit tests itself, didn't think that it belongs to PositionHelper. What I don't like is that this method is repeated in other tests (Highlight, Reference and Rename), should we consider creating a TestBase class? Or is it ok to move this into PositionHelper?

Reply:
If it's repeated, I think it's a good idea to move it to a common place. We can create a test helper class somewhere if we don't have one for position conversion already.

(Btw, we are intentionally not having any base test classes because they make it harder to compose different pieces of test code together and ironically make things harder to reuse.)

Reply:
Moved code into IntegrationTestHelper.